theano/tensor/var.py

```diff
@@ -768,8 +768,7 @@ def choose(self, a, choices, out=None, mode='raise'):
         from.
 
         """
-        return theano.tensor.basic.choose(self, a, choices, out=None,
-                                          mode='raise')
+        return theano.tensor.basic.choose(a, choices, out=None, mode='raise')
```

Comment:
First, I think the signature of the original function is not correct. We should really have:
```python
def choose(self, choices, out=None, mode='raise'):
    return theano.tensor.basic.choose(self, choices, out=out, mode=mode)
```

Second, that means there is no test in the test suite that actually follows this path, otherwise it would have failed.